bin/express

```diff
@@ -312,10 +319,27 @@ function createApplication(app_name, path) {
   });
 }
 
+function addViewEngine(app, viewEngine) {
```

Comment:
The new function should really get a jsdoc block like all the others. And in general, all the functions are organized in alpha-order from top to bottom. Not sure what the thought was for the placement of this function was.